extensions/amp-accordion/1.0/accordion.js

```diff
@@ -331,25 +390,14 @@ export function AccordionSection({
   }, [expanded, animate]);
 
   return (
-    <Comp {...rest} expanded={expanded}>
-      <HeaderComp
-        role="button"
-        className={`${headerClassName} ${classes.sectionChild} ${classes.header}`}
-        aria-controls={contentId}
-        tabIndex="0"
-        onClick={expandHandler}
-        aria-expanded={String(expanded)}
-      >
-        {header}
-      </HeaderComp>
-      <ContentComp
-        ref={contentRef}
-        className={`${contentClassName} ${classes.sectionChild} ${classes.content}`}
-        id={contentId}
-        hidden={!expanded}
-      >
-        {children}
-      </ContentComp>
+    <Comp
+      {...rest}
+      ref={contentRef}
+      className={`${contentClassName} ${classes.sectionChild} ${classes.content}`}
+      id={contentId}
```

Comment:
This part is, unfortunately, more complicated. We need to accept the `id` prop for the component here and when specified, we need to bubble it back up to the section. The section would push back down the `contentId` that both `Header` and `Content` can use. In other words, you need some sort of `setContentId()` in the context along with the existing `contentId`.

Reply:
Sounds good, this will be addressed in follow up PR and is currently tracked in the tracker: https://github.com/ampproject/amphtml/issues/30445

The item is called: `handle protect existing id on content node`